notebooks/1_spikesorting.ipynb

```diff
@@ -4,26 +4,16 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    warnings.simplefilter('ignore', category=DeprecationWarning)
</pre><p>This should not be necessary - just update <code>ipykernel</code>, <code>jupyter</code>, <code>jupyterlab</code> and <code>ipython</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LorenFrankLab/nwb_datajoint/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>A fresh install of nwb_datajoint doesn't have whatever version is necessary to get around this, so I'll keep them in for now. </p>